.sauce/dev.yml

```diff
@@ -28,48 +34,24 @@ npm:
     - "@adobe/reactor-query-string"
     - "@adobe/reactor-promise"
 suites:
-  - name: "Chrome Latest | Windows 11"
+  - name: "Chrome"
     shard: concurrency
-    browserName: "chrome"
-    src:
-      - "test/functional/specs/**/*.js"
     platformName: "Windows 11"
-  
-  - name: "Safari Latest | macOS 12"
-    shard: concurrency
-    browserName: "safari"
+    browserName: "chrome"
     src:
       - "test/functional/specs/**/*.js"
-    platformName: "macOS 12"
-
-  - name: "Edge Latest | Windows 11"
+    excludedTestFiles: [ "test/functional/specs/Privacy/C1576777.js" ]
+  - name: "Edge"
     shard: concurrency
+    platformName: "Windows 11"
     browserName: "microsoftedge"
     src:
       - "test/functional/specs/**/*.js"
-    platformName: "Windows 11"
-
-  - name: "Firefox Latest | Windows 11"
+    excludedTestFiles: [ "test/functional/specs/Privacy/C1576777.js", "test/functional/specs/Personalization/C3272624.js" ]
```

Comment:
why do we exclude these tests?

Reply:
John is currently working on fixing C1576777 and I have a ticket opened for C3272624 failing on Firefox. We need to explicitly state the testcases since the prior release tags still has the tests enabled.